### PR TITLE
test:MM-T125 Copy Link can be used from channel LHS

### PR DESCRIPTION
**Pull request by codedsun**
#### Ticket Link

 Fixes https://github.com/mattermost/desktop/issues/1891

#### Checklist

- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
- [x] executed `npm run lint:js` for proper code formatting

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by codedsun** [comment]
@M-ZubairAhmed @devinbinnie This is passing on my machine, i am afraid what's causing issue in CI.

image

**Comment by codedsun** [comment]
Ok this will not pass as teams and channels are not setup in CI. @devinbinnie @M-ZubairAhmed

File: e2e/specs/mattermost/copy_link.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        await firstServer.waitForSelector('#sidebarItem_off-topic');
+        await firstServer.click('#sidebarItem_off-topic');
+        await firstServer.click('#sidebarItem_off-topic', {button: 'right'});
+        await firstServer.click('div[role="menuitem"]:has-text("Copy Link")');
+        await firstServer.click('#sidebarItem_town-square');
+        await firstServer.click('#post_textbox');
+        const clipboardText = clipboard.readText();
+        await firstServer.fill('#post_textbox', clipboardText);
+        const content = await firstServer.locator('#post_textbox').textContent();
+        content.should.be.equal('/developers/channels/off-topic');
```
Comment (devinbinnie) [inline e2e/specs/mattermost/copy_link.test.js:47]: Just noticed this, the `developers` team won't exist on the test server either, you will have to use the team provided by it (should be `ad-1`, but could be different)

**Comment by codedsun** [comment]
> Yes we do, but I don't think the test user is in the Off-Topic channel by default, so it won't show up in the sidebar. I would try and use Town Square instead

What are the default channels @devinbinnie and how do I see it in code can you refer me?

**Comment by devinbinnie** [comment]
@codedsun You can generate sample data using `mmctl` (https://docs.mattermost.com/manage/mmctl-command-line-tool.html#mmctl-sampledata) and those will be the default channels. Town Square always exists unless changed in the DB so that's the channel you can use here.

**Comment by codedsun** [comment]
@devinbinnie I need 2 channels for this right? Also like do we have this file in the project e2e folder?

**Comment by devinbinnie** [comment]
@codedsun Yes you do, sorry. You can run the sampledata command against your local instance and it'll tell you there.

**Comment by codedsun** [comment]
@devinbinnie Where do I find the file

```
Upload session successfully created, ID: om6ka364otg68kpb8un8hgc37y 
Import file successfully uploaded, name: mmctl-sampledata.zip
Import process job successfully created, ID: qjdm88ybt3dcxmgdjz1fcm3d5o
Sampledata successfully processed
```

**Comment by codedsun** [comment]
@devinbinnie Finally 🎉 

image

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @codedsun!

**Comment by codedsun** [comment]
Thanks a lot team 👍 💯

**Pull request merged**